.storybook/main.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     // Components
     '../projects/canopy/src/lib/accordion/docs/guide.stories.mdx',
     '../projects/canopy/src/lib/accordion/docs/accordion.stories.ts',
+    '../projects/canopy/src/lib/carousel/docs/carousel.stories.ts',
```

Comment:
I had to add the missing storybook story to this list as the Carousel wasn't available to be tested.